scripts/us_census/pep/pep_by_srh/README.md

```diff
@@ -37,21 +37,43 @@ Origin
 
 ## Scripts
 ```
-download.sh
+process.py
 ```
 Above scipt creates necessary folders required for processing files and download's US 
-Census PEP files to folder download_files.
+Census PEP files to folder input_files. And reads downloaded files from input_files folder, prepares consolidated 
+(National, State, County) output files for as-is (population_estimate_by_srh.csv) and aggregated (population_estimate_by_srh_agg.csv) in output_files folder
 
+### Additional Notes
 
-```
-process.py
-```
-Reads downloaded files from download_files folder, prepares consolidated 
-(National, State, County) output files for as-is (population_estimate_by_srh.csv) and aggregated (population_estimate_by_srh_agg.csv) in output_files folder
+We have modified the script for Automation setup, now it's not required to add the future URL's in input_url.jason file. Code will take care of future URL formation automatically.
+
+### Import Procedure
 
+The below command will run process.py and generate output csv, mcf and tmcf. 
+`python3 script/us_census/pep/pep_by_srh/process.py`
 
+Execute the 'process.py' script by using the following commands:
+
+  - if you want to perform "download and process", run the below command:
+
+        `python3 process.py --mode=""`
```

Comment:
Remove the --mode="" argument, as it's the default option and doesn't need to be explicitly specified.

Reply:
Done.